includes/agentic-commerce/class-wc-stripe-agentic-commerce-inventory-tracker.php

```
@@ -101,6 +122,90 @@ public function track_stock_change( \WC_Product $product ): void {
+	public function maybe_track_product_archive( int $post_id ): void {
+		if ( 'product' !== get_post_type( $post_id ) ) {
+			return;
+		}
+
+		$product = wc_get_product( $post_id );
+		if ( ! $product ) {
+			return;
+		}
+
+		$this->track_product_archive( $product );
+	}
```

PHPMD (2.15.0)

[error] The method maybe_track_product_archive is not named in camelCase. (undefined)

(CamelCaseMethodName)